docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -215,11 +215,11 @@ function humsci_basic_preprocess_pattern_vertical_link_card(&$variables) {
     $links = $xpath->query('//a/@href');
     // If the markup is a link, grab the href attribute.
     if ($links->length) {
-      $variables['button'] = $links->item(0)->nodeValue;
+      $variables['href'] = $links->item(0)->nodeValue;
```

Comment:
Did you updated the vertical linked card template to match the new variable name?